Shoalcreek

bnyce opened the pull request.

wbobeirne commented:
For what it's worth, at the end of these instructions I still don't get _quite_ a working website. I'm able to login, I see all the content in the CMS, but the homepage just renders blank. Are there any steps missing that you can see @bnyce?

<img width="1912" alt="Screenshot 2023-10-12 at 10 51 48 AM" src="https://github.com/bnyce/shoalcreek/assets/649992/f22474ed-432e-436b-b5cb-ad97b48db8a9">

bnyce commented:
Hi, thanks for the heads up. I updated the README with instructions and also added a setup.sh file that might be helpful. If you use a different environment than DDEV with colima, you may need to cherry pick what you need...

..I suspect that the issues you encountered were because it was from a not-so-recent db backup and/or the composer update step may have been skipped(?)

wbobeirne commented:
Your setup script and instructions worked great, thanks!
